charts/virtual-kubelet-0.1.1-rbac/templates/deployment.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+    spec:
```

Comment:
add serviceAccountName explicitly.

Reply:
Added serviceAccount from created name in values.yaml